detox/src/utils/customConsoleLogger.js

```diff
@@ -11,7 +11,7 @@ function getStackDump() {
 
 function getOrigin() {
   const userCallsite = callsites()[USER_STACK_FRAME_INDEX];
-  const filename = path.relative(process.cwd(), userCallsite.getFileName());
+  const filename = userCallsite && userCallsite.getFileName() ? path.relative(process.cwd(), userCallsite.getFileName()) : '?';
```

Comment:
Could you split this line into two with
```js
const callsiteFilename = userCallsite && userCallsite.getFileName();`
```
beforehand?

Reply:
For sure, NP.